docs/changelog.md

```diff
@@ -235,30 +261,41 @@ description: |
     arithmetic is needed, such as for finance
   - Added `std` module for accessing standard library definitions even when a
     variable with the same name shadows/overwrites it
+    <!-- I can't find this module in the search. It should have a docs page. -->
   - Added [`array.to-dict`], [`array.reduce`], [`array.windows`] methods
   - Added `exact` argument to [`array.zip`]
   - Added [`arguments.at`] method
   - Added [`int.from-bytes`], [`int.to-bytes`], [`float.from-bytes`], and
     [`float.to-bytes`]
-  - [`calc.round`] no longer accepts negative digits (**Minor breaking change**)
+  - `digits` parameter of [`calc.round`] no longer accepts negative integers
+    (**Minor breaking change**)
   - Conversions from [`int`] to [`float`] will now error instead of saturating
     if the float is too large (**Minor breaking change**)
+    <!-- Pretty sure docs should be updated here:
+         https://staging.typst.app/docs/reference/foundations/float/#constructor
+    -->
   - Added `float.nan` and `float.inf`, removed `calc.nan`
     (**Minor breaking change**)
+    <!-- Docs missing a simple example of printing and/or making these values. -->
   - Certain symbols are now generally callable like functions and not only
     specifically in math. Examples are accents or `floor` and `ceil`.
+    <!-- Add some link/examples.
+         https://staging.typst.app/docs/reference/math/lr#functions-ceil
+         https://staging.typst.app/docs/reference/foundations/calc#functions-ceil
+    -->
   - Improved [`repr`] of relative values, sequences, infinities, NaN,
     `{type(none)}` and `{type(auto)}`
   - Fixed crash on whole packages (rather than just files) cyclically importing
     each other
-  - Fixed behaviour of [`calc.round`] on integers when a non-zero value is
+  - Fixed behavior of [`calc.round`] on integers when a non-zero value is
     provided for `digits`
+    <!-- I wasn't able to reproduce this. -->
```

Comment:
`#type(calc.round(10, digits: 1))` is now `int` instead of `float`

Reply:
Shouldn't then this be called "result type" instead of "behavior"?